configure.ac

```diff
@@ -68,6 +68,11 @@ AC_CHECK_LIB([pthread], [pthread_create], [PTHREAD_LIBS=-lpthread
               AC_DEFINE([HAVE_LIBPTHREAD],,
                         [Whether libpthread was found])])
 
+# librt
+RT_LIBS=
+AC_CHECK_LIB([rt], [timer_create], [RT_LIBS=-lrt],
+             AC_MSG_ERROR("librt is required for performing timer-related operations"))
```

Comment:
Please see https://github.com/apache/guacamole-server/pull/476#discussion_r1447776140:

> It's not _universally_ required. This check should only be fatal where there is no `timer_create()` without linking in librt. If the test determines that `timer_create()` does not require linking with librt, this check should pass and `RT_LIBS` should be empty.

As written, this test will fail the build if librt is absent, even if the platform in question doesn't require librt for `timer_create()`.

Reply:
@mike-jumper I've fixed the above code specified by you.

Reply:
@mike-jumper I've fixed the above code specified by you.